admin_guide/out_of_resource_handling.adoc

```diff
@@ -298,7 +540,7 @@ those pods first.
 
 [NOTE]
 ====
-At this time, volumes that are backed by local disk are only deleted when a pod
+Volumes that are backed by local disk are only deleted when a pod
```

Comment:
> Volumes that are backed by local disk are only deleted when a pod is deleted from the API server instead of when the pod is terminated.

This is not true as of 1.6.  They are deleted when the pod is terminated.

Probably just need to drop this whole note section now.